packages/client/src/BrubeckNode.ts

```diff
@@ -228,12 +228,39 @@ export default class BrubeckNode implements Context {
 
     async closePublishProxyConnectionOnStreamPart(streamPartId: StreamPartID, nodeId: string): Promise<void> {
         try {
-            if (!this.cachedNode || !this.startNodeComplete) {
+            if (this.isStarting()) {
                 return
             }
             await this.cachedNode!.leavePurePublishingStreamPart(streamPartId, nodeId)
         } finally {
             this.debug('closeProxyConnectionOnStream << %o', streamPartId, nodeId)
         }
     }
+
+    async openSubscribeProxyConnectionOnStreamPart(streamPartId: StreamPartID, nodeId: string): Promise<void> {
+        try {
+            if (this.isStarting()) {
+                await this.startNodeTask()
+            }
+            await this.cachedNode!.joinStreamPartAsPureSubscriber(streamPartId, nodeId)
+        } finally {
+            this.debug('openProxyConnectionOnStream << %o', streamPartId, nodeId)
+        }
+    }
+
+    async closeSubscribeProxyConnectionOnStreamPart(streamPartId: StreamPartID, nodeId: string): Promise<void> {
+        try {
+            if (this.isStarting()) {
+                return
+            }
+            await this.cachedNode!.leavePureSubscribingStreamPart(streamPartId, nodeId)
+        } finally {
+            this.debug('closeProxyConnectionOnStream << %o', streamPartId, nodeId)
+        }
+    }
+
+    private isStarting(): boolean {
```

Comment:
Name is a bit confusing cause it seems to refer to something actively starting when it returns true. Would prefer something that indicates that node needs starting, e.g.
```suggestion
    private nodeRequiresStarting(): boolean {
```

or flipping the logic 
```suggestion
    private hasNodeStarted(): boolean {
```